### PR TITLE
Add container sizes

### PR DESCRIPTION
**Pull request by daniarsuprianto**

### CODE REVIEW

**Comment by jpsc** [comment]
Hello @daniarsuprianto,

Since we have different values for the default breakpoints, have thought about updating the [container sizes](https://tailwindcss.com/docs/container) instead of creating new classes?

**Comment by daniarsuprianto** [comment]
> Hello @daniarsuprianto,
> 
> Since we have different values for the default breakpoints, have thought about updating the [container sizes](https://tailwindcss.com/docs/container) instead of creating new classes?

hey.. yes i've thought about that. the problem with tailwind's container class is it responds to the screen size. so based on breakpoint, it will have different max width. i added width here because it doesnt change for different breakpoints. maybe i misunderstand the `container` class on tailwind. we can talk about it on FE cluster today

**Comment by jpsc** [comment]
No, you understood it correctly. My question is why would you need for example to set a container size small if you are not on a small screen?

File: index.js [hunk 1]
```diff
@@ -79,6 +79,12 @@
             maxWidth: {
                 '1/3': '33.333%',
                 '2/3': '66.666%',
+                container: '984px',
+                'container-xs': '480px',
```
Comment (dine) [inline index.js:83]: Do we need all the variations? I believe we are using currently `container` and `container-lg`

**Review by dine — APPROVED** [review]

**Comment by daniarsuprianto** [comment]
>No, you understood it correctly. My question is why would you need for example to set a container size small if you are not on a small screen?

@jpsc For now I only added one size, we indeed dont need the other sizes :)

File: index.js (hunk 1 above)
Comment (daniarsuprianto) [reply index.js]: removed!

**Pull request merged**